viam-cartographer/cartographer_patches/carto.patch

```diff
@@ -0,0 +1,304 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
This file gets applied on top of the upstream cartographer submodule during ensure-submodule-initialization. Changes within that submodule can thus be made "permanent" with `git diff --output ...` set to this file.